go/zeusclient/zeusclient.go

```diff
@@ -152,6 +174,21 @@ func Run(args []string, input io.Reader, output *os.File) int {
 		}
 	}()
 
+	eofStderr := make(chan bool)
+	go func() {
+		for {
+			buf := make([]byte, 1024)
+			n, err := masterStderr.Read(buf)
+
+			if err == nil || (err == io.EOF && n > 0) {
+				stderr.Write(buf[:n])
+			} else {
+				eofStderr <- true
+				break
+			}
+		}
+	}()
```

Comment:
There is a lot of duplication here, should create seperate functions of doing setup and read and then just pass in the different streams...